contracts/SportMarkets/SportsAMM.sol

```diff
@@ -158,22 +160,27 @@ contract SportsAMM is Initializable, ProxyOwned, PausableUpgradeable, ProxyReent
     SportAMMLiquidityPool public liquidityPool;
 
     // @return specific min_spread per address
-    mapping(uint => mapping(uint => uint)) public minSpreadPerSport;
+    mapping(uint => mapping(uint => uint)) public minSpreadPerSport; //deprecated see SportAMMRiskManager.sol
 
     /// @return the sport which is one-sider
     mapping(uint => bool) public isMarketForSportOnePositional;
```

Comment:
can this also be moved to Risk Manager, but then you need to add RiskManager into SportAMMUtils because this is called there?

1) This is some risk management property more or less
2) Player props can also be set up as one-siders so we need to extend that

Reply:
yes, moved